modules/pricer/src/main/java/com/opengamma/strata/pricer/deposit/DiscountingIborFixingDepositProductPricer.java

```diff
@@ -125,14 +128,18 @@ public PointSensitivities parSpreadSensitivity(IborFixingDepositProduct product,
   // query the forward rate
   private double forwardRate(ExpandedIborFixingDeposit product, ImmutableRatesProvider provider) {
     IborIndex index = product.getFloatingRate().getIndex();
-    IborIndexRates rates = provider.iborIndexRates(index);
+    Curve curve = provider.getIndexCurves().get(index);
+    IborIndexRates rates = DiscountIborIndexRates.of(index, LocalDateDoubleTimeSeries.empty(),
```

Comment:
After a bit of thinking, I can see what this change does - i.e. by passing in an empty fixing series, the rate will always be sourced from the forward curve, whereas previously it implicitly uses the fixing, if available, in preference to the forward curve. This is quite subtle, so I think it would be worth a comment here to explain this to others.